src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/hmi_notifications_test.cc

```diff
@@ -192,6 +192,14 @@ const int32_t kMobileProtocolType_ = 0;
 const int32_t kProtocolVersion_ = 3;
 const uint32_t kCorrelationId_ = 1939u;
 const uint32_t kAppId_ = 2014u;
+const std::string kDefaultLanguage = "en-us";
+const mobile_apis::Language::eType kMobileLanguage =
+    mobile_apis::Language::EN_US;
+
+// LSDW - lock screen dismissal warning
+const std::string kLSDW_message_en =
```

Comment:
@IGapchuk LSDL? typo or strange abbreviature ?

Reply:
@LuxoftAKutsan Fixed in 1453eb695bf77457789f9b9260b0a365926eaa39